waypoints: [[0.656, 0.023, 1.564], [0.656, 2.189, -0.003], [3.581, 2.287, 1.533], [3.581, 2.949, 0.007], [5.736, 2.895, -1.6], [5.736, -0.189, -1.542]]

#waypoints: [[0.754, -0.053, -0.064], [0.754, 2.020, 0.000], [3.9, 2.017, 0.020], [3.9, 2.728, 0.072], [5.872, 2.728, -0.035], [5.872, -0.046, -1.601]]

#waypoints: [[0.0, 2.0, 0.029], [2.0, 2.0, -0.064], [0.0, 0.0, 0.000], [2.0, 0.0, 0.020], [1.0, 1.0, 0.072], [3.0, 1.0, -0.035]]

#adj_pairs: [[0, 4], [2, 4], [1, 4], [3, 4], [3, 5], [1, 5]]
adj_pairs: [[0, 1], [1, 2], [2, 3], [3, 4], [4, 5]]
